manifests/d/drakkan/SFTPGo/v2.4.2/drakkan.SFTPGo.locale.en-US.yaml

```
@@ -0,0 +1,30 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: drakkan.SFTPGo
+PackageVersion: v2.4.2
+PackageLocale: en-US
+Publisher: Nicola Murino
+PackageName: SFTPGo
+PackageUrl: https://github.com/drakkan/sftpgo
+License: AGPL-3.0
+Copyright: AGPL-3.0
+LicenseUrl: "https://github.com/drakkan/sftpgo/blob/main/LICENSE"
```

```suggestion
LicenseUrl: https://github.com/drakkan/sftpgo/blob/main/LICENSE
```
This doesn't need to be in speech marks